### ISSUE TITLE
FlxScrollZone - non-alpha?

### ISSUE THREAD

**Issue by SeiferTim** [photonstorm/Flixel-Power-Tools#1]
ScrollZone doesn't work too well if you're messing with alpha transparencies.
To fix, in the scroll function, right I set data.source.pixels to a new, empty, transparent BitmapData. It seems to work without any speed/performance problems (but I'm using a tiny Png).

This is my 'fix':

`````` ...
zone.buffer.graphics.endFill();
data.source.pixels = new BitmapData(data.source.pixels.width, data.source.pixels.height, true, 0x00000000); /// <- added
data.source.pixels.draw(zone.buffer, zone.drawMatrix);
...```
``````

**Comment by photonstorm** [comment]
Thanks - this is fixed in FlxScrollZone v1.4 which I'll push to dev tonight. I've done it a little different to the way you suggested (I use bitmapData.fillRect rather than create a new bitmapData every loop) but it works just the same! It can be applied to the whole image or just one scroll zone.

**Comment by photonstorm** [comment]
This is now fixed in v1.6 Beta 1 which is up on the dev branch.
